editor-components/feedback/feedbackmodal1/feedbackmodal1.tsx

```diff
@@ -0,0 +1,249 @@
+import * as React from "react";
+import { BaseModal } from "../../EditorComponent";
+import styles from "./feedbackmodal1.module.scss";
+import { ErrorMessage, Formik, Form, Field } from "formik";
+import * as Yup from "yup";
+
+type Emoji = {
+  id: number;
+  label: string;
+  icon: string;
+  color: string;
+};
+
+const emojis: Emoji[] = [
+  { id: 1, label: 'Very Bad', icon: '😢', color: '#E74C3C' },
+  { id: 2, label: 'Bad', icon: '😞', color: '#F39C12' },
+  { id: 3, label: 'Medium', icon: '😐', color: '#F1C40F' },
+  { id: 4, label: 'Good', icon: '😊', color: '#2ECC71' },
+  { id: 5, label: 'Very Good', icon: '😂', color: '#3498DB' },
+];
```

Comment:
icon cannot be an emoji. also remove this array. add this prop hard coded into the constructor. remove the map.